src/types/address.rs

```diff
@@ -85,23 +85,6 @@ impl<'de> Deserialize<'de> for Address {
     }
 }
 
-impl Address {
-    pub fn to_address(&self) -> BitcoinAddress {
-        self.0.clone()
-    }
-
-    // TODO: trash this method in favor of to_address
-    pub fn to_bitcoin_address(&self) -> Result<BitcoinAddress, bitcoin::util::Error> {
-        Ok(self.0.clone())
-    }
-}
-
-impl fmt::Display for Address {
```

Comment:
Any reason why the `Display` implementation is removed as well?

Reply:
ups, good find 😇

Reply:
Could add a test to assert this functionality :)